modules/gcp_k8s_base/gcp-k8s-base.yaml

```diff
@@ -71,6 +71,21 @@ inputs:
     validator: bool(required=False)
     description: Expose self-signed SSL certs.
     default: false
+  - name: cert_manager_values
+    user_facing: true
+    validator: any(required=False)
+    description: Certificate Manager helm chart additional values. [Available options](https://artifacthub.io/packages/helm/cert-manager/cert-manager?modal=values)
+    default: {}
```

Comment:
I don't think we should change cert manager since we want to remove it https://run-x.atlassian.net/browse/RUNX-1216

if some users use this new values then we won't be able to remove it.